Monika After Story/game/script-ch30.rpy

```diff
@@ -2140,4 +2140,8 @@ label ch30_reset:
             if not mas_inEVL("mas_consumables_remove_thermos"):
                 queueEvent("mas_consumables_remove_thermos")
 
+    # Delete .chr files on startup
+    if mas_seenLabels(["mas_new_character_file"]):
```

Comment:
Looks like your branch is severely outdated, please update it and resolve the conflicts.

Reply:
@dreamscached you will need to use the new reset system #8745

Reply:
Oh right, thank you, I will!